### PR TITLE
Adds data fetch scheduler dag

### PR DESCRIPTION
**Pull request by justinlittman**

### CODE REVIEW

File: docker-compose.prod.yaml [hunk 1]
```diff
@@ -78,6 +78,7 @@ x-airflow-common:
     AIRFLOW_VAR_FOLIO_USER: ${AIRFLOW_VAR_FOLIO_USER}
     AIRFLOW_VAR_FOLIO_PASSWORD: ${AIRFLOW_VAR_FOLIO_PASSWORD}
     AIRFLOW_VAR_OKAPI_URL: ${AIRFLOW_VAR_OKAPI_URL}
+    AIRFLOW_CONN_VENDOR_LOADS: postgresql+psycopg2://airflow:airflow@postgres/vendor_loads
```
Comment (aaron-collier) [inline docker-compose.prod.yaml:81]: I think this should use the DATABASE_USERNAME, DATABASE_PASSWORD, and DATABASE_HOSTNAME env vars as seen on line 60.

Comment (justinlittman) [reply docker-compose.prod.yaml]: Ooops. Cut and paste error. Fixed.

**Review by aaron-collier — APPROVED** [review]

**Review by ndushay — COMMENTED** [review]
methinks we should standardize on formatting.  A lot of this is lint noise, and could get changed back by another developer.

**Comment by justinlittman** [comment]
@ndushay My understanding is that the standard is to run https://github.com/psf/black for formatting.

**Pull request merged**